### DISCUSSION TITLE
[Unstable] Hull-name character limit breaks (most) localization strings.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#14547] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Weirdly, in this latest unstable update hull names are now limited to 20 characters all of a sudden.

This is actually so short that it breaks many of the game's default hull names.
![image](https://github.com/user-attachments/assets/10b15f84-527b-419b-b44c-d6a9760951cb)
![image](https://github.com/user-attachments/assets/eb7c127b-f6eb-4205-ba2d-fd3833042f36)
![image](https://github.com/user-attachments/assets/f5bd39c0-9503-4aab-a5ae-a146652dd79f)
![image](https://github.com/user-attachments/assets/8fbec560-4bf7-4fcb-8846-d405c7cea773)

### Version

v1.6.4.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been fixed in our private repo now, fix coming up in the next unstable build. Sorry for the inconvenience!
